Classes/JTLocalizeUtils.m

```diff
@@ -106,7 +117,16 @@ + (void)setLocalizationBundleToPath:(NSString *)bundlePath stringsTableName:(NSS
 }
 
 + (NSString *)effectiveLocale {
-    return [self instance].effectiveLocale;
+    NSString *effectiveLocale = [self instance].effectiveLocale;
+    if ([[[JTGameConfig sharedInstance] settingForKey:@"dutchEnabled" defaultValue:@(false)] boolValue]) {
+        return [self instance].effectiveLocale;
+    } else {
+        if ([effectiveLocale isEqualToString:@"nl"]) {
```

Comment:
Shouldn't this be impossible because of the other function that prevents effective locale to be set as `nl` on the instance?

Reply:
You are probably right, I guess I can change this but tested already with it. After the test this will go one way or another so not changing it now. Good catch